docs/self-managed/operational-guides/update-guide/870-to-880.md

```diff
@@ -0,0 +1,20 @@
+---
+id: 870-to-880
+title: Update 8.7 to 8.8
+description: "Review which adjustments must be made to migrate from Camunda 8.7.x to Camunda 8.8.0."
+---
+
+The following sections explain which adjustments must be made to migrate from Camunda 8.7.x to 8.8.x for each component.
+
+## Removed deprecated OpenAPI objects
+
+In previous releases we gradually transformed our entity keys from `number (int64)` to `string` types while still
+providing users the opportunity to keep using the deprecated versions. However, with 8.8 release we removed the already
+deprecated API objects containing number keys and with that, the additional content type headers, too.
+
+If you already switched to `string` keys, then this change will not affect you in any way.
```

Comment:
🔧 I guess it could affect users if they use the explicit string media type header (since 8.6, for example, to fix a former 8.6 client application).